tests/resources/installrdf/shouldnt_exist.rdf

```diff
@@ -8,6 +8,7 @@
 	<em:type>2</em:type>
 	
 	<em:updateURL>true</em:updateURL>
+	<em:updateKey>meh</em:updateKey>
```

Comment:
We just never checked for it before. I added it because we're now warning when it's missing, for unhosted add-ons.